prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -68,19 +74,26 @@ def smart_charging(
     :return: (*numpy.ndarray*) -- charging profiles.
     """
 
+    # for code developer debugging printouts (1 = yes, 0 = no)
+    debug_printout = 0
+
     # load NHTS data from function
     if veh_type.lower() == "ldv":
-        newdata = data_helper.remove_ldt(data_helper.load_data(census_region, filepath))
+        newdata1 = data_helper.remove_ldt(
```

Comment:
Why use `newdata1` as variable name?

Reply:
That was something I did in case the original dataframe shouldn't be updated via the filtering process we added.
Now that we have automated tests, **and** as long as my concern is outdated, then I'm on board with only using `newdata` and not creating `newdata1`